index.html

```diff
@@ -778,46 +802,6 @@ <h3>Signatures</h3>
 <a>cryptographic suite</a> to produce a verifiable digital signature.
         </p>
 
-<div class="issue">Create a separate section detailing an optional mechanism
```

Comment:
Everything below here is deleted because it was duplicated content. All of this text still exists elsewhere in the spec.